docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,56 @@
+# Contribution to Tartiflette
+
+:tada: Thanks for being here :tada:
+
+* [Code of Conduct](#code-of-conduct)
+* [How can I contribute?](#how-can-i-contribute)
+  * [Reporting Bugs](#reporting-bugs)
+  * [Suggesting Enhancements](#suggesting-enhancements)
+  * [Pull Requests](#pull-requests)
+* [Changelog](#changelog)
+
+## Code of conduct
+
+We expect from each contributor a respect for the [Code of Conduct](./CODE-OF-CONDUCT.md) which governs the **Tartiflette** project.
+
+## How can I contribute?
+### Reporting bugs
+
+To report a bug, you have to [create an issue on github](https://github.com/dailymotion/tartiflette/issues/new) and follow the following guidelines.
+
+Please provide the steps to reproduce and if possible a reproductible environment. **As we are working directly with container, please provide the Dockerfile sample or the Docker image name**
+
+* [ ] **Tartiflette version:** _e.g 0.1.0_
+* [ ] **Python version:** _e.g 3.6_
+* [ ] **Executed in docker:** _Yes_
+* [ ] **Dockerfile sample:** _Link of sample_
+* [ ] **GraphQL Schema & Query:** _gist, pastebin or directly the query_
+* [ ] **Explain with a simple sentence the behavior expected**
+* [ ] **Is a regression from a previous versions?** _e.g Yes_
+
+### Suggesting enhancements
+
+* A feature is missing from the GraphQL Specification?
+* A hook is missing in the API to plug your code?
+* A part of the project could be improved?
+
+Don't hesitate to ask a new feature or code it by yourself.
```

Comment:
Don't hesitate to ask a new feature or join in and send us a pull request with your own feature !